package.json

```diff
@@ -18,7 +18,7 @@
   },
   "scripts": {
     "deploy:prod": "DEV=0 NODE_ENV=production npm run build && surge build --domain rloop.org",
-    "deploy:dev": "npm run build && surge build --domain https://rloop-beta.surge.sh",
```

Comment:
Please don't change this. (You shouldn't even need to change it for development, just use `npm run dev`.